script.js

```diff
@@ -0,0 +1,63 @@
+// navigation
+let navLink = document.querySelectorAll('nav a');
+
+// main section
+let mainContent = document.querySelectorAll('.main-content');
+let services = document.querySelectorAll('.services');
+let product = document.querySelectorAll('.product');
+let vision = document.querySelectorAll('.vision');
+let features = document.querySelectorAll('.features');
+let about = document.querySelectorAll('.about');
+let contact = document.querySelectorAll('.contact');
+let reset = document.querySelectorAll('.reset');
+
+const resetBackground = function () {
+  // // reset nav links
+  // navLink.forEach(el => {
+  //   el.style.backgroundColor = 'initial';
+  // })
+  // // reset text content
+  // textContent.forEach(el => {
+  //   el.style.backgroundColor = 'initial';
+  // })
+  // // reset contact
+  // contact.forEach(el => {
+  //   el.style.backgroundColor = 'initial';
+  // })
+  reset.forEach(el => {
```

Comment:
Because we now must call the function it adds work but I am always a fan of encapsulation.

Notice though that the reference to `reset` is defined outside of resetBackground. This can and should be moved inside.